ios/xcode/MGLKit/MGLKViewController+iOS.mm

```diff
@@ -47,6 +50,8 @@ - (void)resume
     [self pause];
     NSLog(@"MGLKViewController resume");
 
+    _isPaused = NO;
```

Comment:
I think there should be a check to early exit if `_isPaused` is already false.
For example, if user set `pauseOnWillResignActive` to false, but keep `_resumeOnDidBecomeActive` to true, there will be a redundant resume.

Reply:
thanks for taking a look a this :), I've added early exits for pause and resume and move setting the bool to the end (this is so it's only set when the call succeeds; resume can early exit if glView or glView.window is null)